code/edge_detection.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>Would using the gaussian blur beforehand also help here to get rid of the noise inside the image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>